highAndLow.js

```diff
@@ -0,0 +1,28 @@
+/*
+In this little assignment you are given a string of space separated numbers, and have to return the highest and lowest number.
+
+Examples
+highAndLow("1 2 3 4 5");  // return "5 1"
+highAndLow("1 2 -3 4 5"); // return "5 -3"
+highAndLow("1 9 3 4 -5"); // return "9 -5"
+
+Notes
+All numbers are valid Int32, no need to validate them.
+There will always be at least one number in the input string.
+Output string must be two numbers separated by a single space, and highest number is first.
+*/
+
+function highAndLow(numbers){
+  let arrayOfStrings = numbers.split(" ");
+  
+  let arrayOfNumbers = arrayOfStrings.map(string => {
```

Comment:
Jedna pętla zamiast dwóch.

Plus zerknij na Math.max/min + JS apply method / spread operator i będziesz miał krótsze rozwiązanie :)

Reply:
Hmm dobra zrobiłem coś takiego, ale możemy przegadać dokładniej to apply? bo zrobiłem, ale chyba nie do końca rozumiem jak to działa